tests/components/orderentry/SortAndFilter.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import SortAndFilter from '../../../app/js/components/orderEntry/SortAndFilter';
+import sortAndFilterAction from '../../../app/js/actions/sortAndFilterAction';
+
+const store = mockStore({})
+const props = {
+  sortAndFilterAction: jest.fn()
+}
+const wrapper = shallow(<SortAndFilter store={store} />).dive();
+
+describe('sort and Filter component test-suite', () => {
+  it('renders properly', () => {
+    const selectElement = wrapper.find('select').at(0)
+    selectElement.simulate('change', { target: { value: 'testorder' }});
+    expect(wrapper.exists()).toBeTruthy();
+    expect(selectElement.exists()).toBeTruthy();
+  });
+});
```

Comment:
After simulating change with:
```
selectElement.simulate('change', { target: { value: 'testorder' }});
```
the component is supposed to call a function with some arguments, please add test for that.

Reply:
addressed